ci/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:latest
+FROM registry.fedoraproject.org/fedora:32
```

Comment:
Let's change this to `fedora:33`, but otherwise looks good.

Reply:
33 is what it initially had, but that doesn't work. See the commit message...